app/fastapi_app/routers/sensors.py

```
@@ -0,0 +1,716 @@
+        "has_prev": page > 1
+    }
+
+
+# ============================================================================
+# HTML Pages (Legacy Support)
+# ============================================================================
+
+@router.get("/", name="sensors_list_page")
+async def sensors_list_page(
+    request: Request,
+    db: AsyncSession = Depends(get_async_db),
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)